objectivec/DevTools/compile_testing_protos.sh

```diff
@@ -25,6 +25,64 @@ case "${ACTION}" in
     ;;
 esac
 
+# -----------------------------------------------------------------------------
+# Most of the proto files that exist in the C++ src tree.  Several are used in
+# the tests, but the extra don't hurt in that they ensure ObjC sources can be
```

Comment:
Could you clarify the second sentence in the comment? For some reason I'm having troubles parsing what you mean.

Reply:
tweaked.